category_encoders/hashing.py

```diff
@@ -101,9 +102,10 @@ class HashingEncoder(util.BaseEncoder, util.UnsupervisedTransformerMixin):
     """
     prefit_ordinal = False
     encoding_relation = util.EncodingRelation.ONE_TO_M
+    default_int_np_array = np.array(np.zeros((2,2), dtype='int'))
```

Comment:
why is this 2x2?

Reply:
Im not sure 😅 I am just using it to get the type of the default int array name in [that line](https://github.com/scikit-learn-contrib/category_encoders/pull/428/files/206de2d8327489c4ff4d2c7c4f566c0fc06210c1#diff-5871b042f65ccab77377b2e9a92ea2c9651cc039b020835b6d77bfcb01ffe475R187) so it could be 1 by 1!